src/electron/build_icons.action.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#
+# Copyright 2018 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -eux
+
+electron-icon-maker $@
```

Comment:
so `electron-icon-maker` has no api, but there is an issue open to change that - https://github.com/jaretburkett/electron-icon-maker/issues/2

our options are to make a pr (which hoooonestly wouldn't be too much work) or to find a more modern alternative. i'll follow up on this later, unless y'all have any better ideas

Reply:
Another option is to generate the icons on Linux/Mac, and check them in.

Reply:
Could maybe include those in the `src/electron` folder, too, yeah.

Reply:
Done, works great!